manifests/install.pp

```diff
@@ -1,22 +1,18 @@
+# @summary Set a kernel module as installed
 #
-# == Definition: kmod::install
-#
-# Set a kernel module as installed.
-#
-# Parameters:
-# - *ensure*: present/absent;
-# - *command*: optionally, set the command associated with the kernel module;
-# - *file*: optionally, set the file where the stanza is written.
-#
-# Example usage:
+# @param ensure State of the setting
+# @param command Command associated with the kernel module
+# @param file File where the stanza is written
 #
+# @example
 #   kmod::install { 'pcspkr': }
-#
 define kmod::install (
-  $ensure=present,
-  $command='/bin/true',
-  $file="/etc/modprobe.d/${name}.conf",
+  Enum['present', 'absent'] $ensure  = 'present',
+  String[1]                 $command = '/bin/true',
+  Stdlib::Unixpath          $file    = "/etc/modprobe.d/${name}.conf",
 ) {
+  include kmod
```

Comment:
why did you add the include here?

Reply:
We need that to make sure that `/etc/modprobe.d` is managed.